addons/building_markers/functions/fnc_set.sqf

```diff
@@ -61,6 +61,8 @@ if (_set) then {
     _marker setMarkerDir getDir _object;
     _object setVariable [QGVAR(marker), _marker, true];
 
+    [QEGVAR(common,setMarkerDrawPriority), [_marker, -1]] call CBA_fnc_globalEvent;
```

Comment:
How does this work for JIP?

Reply:
Ah! Good call. Definitely did not work JIP.
Fix works on testing with loopback clients.